### ISSUE TITLE
Add Monaspace Nerd Font

### ISSUE THREAD

**Issue by redactedscribe** [matthewjberger/scoop-nerd-fonts#260]
Not sure if new fonts are automatically added, but I'd like to see Monaspace/Monaspice in the bucket once it's available: https://github.com/ryanoasis/nerd-fonts/pull/1405

Thanks.

**Comment by jonz94** [comment]
Thanks for bringing this information!

We rely on the upstream nerd-fonts repository put the patched font in their [release page](https://github.com/ryanoasis/nerd-fonts/releases),
because what the font installer in this repository do is download the font files from their release page.

That means, after nerd-fonts releases a newer version, we can easily add `Monaspace` & `Monaspice` into the bucket.

**Comment by redactedscribe** [comment]
No problem.

> `Monaspace` font already added in #259

This is the original Monaspace (non-patched) and the reason why I ask for the Nerd Fonts version.

I will wait until Monaspice is present in a NF release.

**Comment by EphemeralDev** [comment]
Monaspace should be available with the latest [release](https://github.com/ryanoasis/nerd-fonts/releases/tag/v3.1.0)

**Comment by redactedscribe** [comment]
Thanks @jonz94!

I've usually just downloaded and used the `*-NF-Mono` fronts available in this bucket and it's been fine. But I'm curious, what are (and why would I want) the plain `-NF` or `NF-Propo` variants?

**Comment by jonz94** [comment]
Great question!

> I think this might be to do with the flags the fonts have set, right? Some text editors will only let your pick a monospaced font if it's a "true" monospaced font, which I'm guessing only the `*-NF-Mono` variant is.

Yes! You're right! The `*-NF-Mono` variant serves this purpose.

And the full description of each font can be seen using the `scoop info` command:

```
PS C:\> scoop info Monaspace-NF

Name        : Monaspace-NF
Description : Nerd Fonts patched 'Monaspace' Font family. (Normal version, Nerd Fonts Symbol/Icon could be 1 or 2 cell wide)
Version     : 3.1.0
Bucket      : nerd-fonts
Website     : https://github.com/ryanoasis/nerd-fonts
License     : MIT
Updated at  : 11/22/2023 2:17:55 AM
Updated by  : jonz94

PS C:\>scoop info Monaspace-NF-Mono

Name        : Monaspace-NF-Mono
Description : Nerd Fonts patched 'Monaspace' Font family. (Monospace version, Nerd Fonts Symbol/Icon will be always 1 cell wide)
Version     : 3.1.0
Bucket      : nerd-fonts
Website     : https://github.com/ryanoasis/nerd-fonts
License     : MIT
Updated at  : 11/22/2023 2:17:55 AM
Updated by  : jonz94

PS C:\> scoop info Monaspace-NF-Propo

Name        : Monaspace-NF-Propo
Description : Nerd Fonts patched 'Monaspace' Font family. (Proportional version, for GUI usecases)
Version     : 3.1.0
Bucket      : nerd-fonts
Website     : https://github.com/ryanoasis/nerd-fonts
License     : MIT
Updated at  : 11/22/2023 2:17:55 AM
Updated by  : jonz94
```

**Comment by redactedscribe** [comment]
Thanks!

**Comment by jonz94** [comment]
Resolved by #270
